packages/api/tests/models/test-rules-model.js

```diff
@@ -273,3 +273,17 @@ test.serial('Creating a kinesis rule where an event source mapping already exist
     lambdaStub.reset();
   }
 });
+
+// This test currently fails because queueName is not an acceptable top
+// level parameter in the rule schema. When the work is done to change
+// the dashboard, that should be updated.
+test.skip('Creating a rule with a queueName parameter', async (t) => {
```

Comment:
As the comment suggests, this test will always fail because the schema hasn't been updated to include `queueName`. I'm open to suggestions - is this something I should add as part of this ticket? Is this something I can kick down the road to the person who does the updates to dashboard?

Reply:
It seems just as easy to do it now, unless there is a specific reason not to?